src/main/java/ch/njol/skript/tests/platform/Environment.java

```diff
@@ -235,7 +235,9 @@ public TestResults runTests(Path runnerRoot, Path testsRoot, boolean devMode, bo
 			args.add("-Dskript.forceregisterhooks");
 		args.add("-Dskript.testing.results=test_results.json");
 		args.add("-Ddisable.watchdog=true");
-		args.addAll(Arrays.asList(jvmArgs));
+		if (debug)
+			args.add("-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=1044");
```

Comment:
Is there a reason you're using port 1044? It's typically 8000.

Reply:
Doesn't matter all that much but I'll change it to 8000.